lib/db/mysql/patches/patch-014-015.sql

```diff
@@ -0,0 +1,50 @@
+-- Add purge expired tokens stored procedure and index to expiresAt column
+ALTER TABLE tokens ADD INDEX idx_expiresAt (expiresAt);
```

Comment:
When using Travis, `ALTER TABLE tokens ADD INDEX idx_expiresAt (expiresAt), ALGORITHM = INPLACE, LOCK = NONE;` causes the tests to fail with 

`fxa-oauth-server.db.mysql.ERROR: updateDbSchema Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ALGORITHM = INPLACE, LOCK = NONE;`

I think this might be related to the version of mysql running in Travis because it passes locally with 5.6.

@jrgm do you know which version of mysql is running in aws?